NuGet.Docs/docs/Get-Started/Install-NuGet.md

```diff
@@ -1,13 +1,112 @@
-#Install NuGet
+#NuGet Tools
+
+Whether consuming or creating NuGet packages you can use the following tools to meet your needs:
+
+##NuGet.exe Command-line Tool
+This utility can be used to create, publish, and download packages. 
+
+How do I get NuGet.exe?
+<ol>
+<li>1. Download the latest version from here and save the file to a path of your choice.</li>
+<li>2. Add this path to the PATH Environment Variable.</li>
+</ol>
+
+<div class="block-callout-warning">
+	<strong>NuGet 2.x users</strong><br>
+    Because there are a few breaking changes introduced in NuGet 3.2 https://nuget.org/nuget.exe is pointing to the NuGet 2.x latest release, to prevent CI systems from auto upgrading and potentially breaking.
```

Comment:
Because there are a few breaking changes introduced in NuGet 3.2, https://nuget.org/nuget.exe points to the latest stable NuGet 2.x release to prevent CI systems from potentially breaking at this time.